CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/ConversationExtensions.cs

```diff
@@ -61,11 +65,162 @@ public static async Task<TeamsChannelAccount[]> GetTeamsConversationMembersAsync
+            var teamsPagedMembersResult = new TeamsPagedMembersResult
+            {
+                Members = pagedMembersResult.Members.Select(member => member.AsTeamsChannelAccount()).ToArray(),
+                ContinuationToken = pagedMembersResult.ContinuationToken
+            };
+            return teamsPagedMembersResult;
+        }
+
+        /// <summary>
+        /// GET Paged teams conversation members asynchronously.
+        /// </summary>
+        /// <param name="conversations">Conversation instance.</param>
+        /// <param name="client">Connector Client</param>
+        /// <param name='userId'> Conversation ID </param>
+        /// <param name='conversationId'> Conversation ID </param>
+        /// <param name='customHeaders'> Headers that will be added to request.</param>
+        /// <param name='cancellationToken'> The cancellation token. </param>
+        /// <returns>Returns the request member.</returns>
+        public static async Task<TeamsChannelAccount> GetTeamsConversationMemberAsync(this IConversations conversations, ConnectorClient client, string userId, string conversationId, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            using (var getMemberResponse = await GetConversationMemberWithHttpMessagesAsync(client, userId, conversationId, customHeaders: customHeaders, cancellationToken: cancellationToken).ConfigureAwait(false))
+            {
+                var member = getMemberResponse.Body;
+                return member.AsTeamsChannelAccount();
+            } 
+        }
+
+        /// <summary>
+        /// GET single conversation member.
+        /// </summary>
+        /// <param name="Client">Bot framework connector client</param>
+        /// <param name='userId'> Conversation ID</param>
+        /// <param name='conversationId'> Conversation ID</param>
+        /// <param name='customHeaders'> Headers that will be added to request. </param>
+        /// <param name='cancellationToken'> The cancellation token. </param>
+        /// <exception cref="ErrorResponseException"> Thrown when the operation returned an invalid status code </exception>
+        /// <exception cref="SerializationException"> Thrown when unable to deserialize the response </exception>
+        /// <exception cref="ValidationException"> Thrown when a required parameter is null </exception>
+        /// <exception cref="System.ArgumentNullException"> Thrown when a required parameter is null </exception>
+        /// <return> A response object containing the response body and response headers. </return>
+        public static async Task<HttpOperationResponse<ChannelAccount>> GetConversationMemberWithHttpMessagesAsync(this ConnectorClient Client, string userId, string conversationId, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
just curious, what does WithHttpMessages mean/indicate to the developer?

Reply:
This method is similar to those existing in v3 bf, therefore just wanted to follow naming convention for BF connector methods; that said you're correct that it could potentially be confusing to the devs